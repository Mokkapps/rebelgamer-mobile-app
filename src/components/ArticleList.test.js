import React from 'react';
import renderer from 'react-test-renderer';

import ArticleListScreen from './ArticleListScreen';

it('renders correctly', () => {
  const navigation = { navigate: jest.fn() };
  const tree = renderer
    .create(<ArticleListScreen navigation={navigation} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
