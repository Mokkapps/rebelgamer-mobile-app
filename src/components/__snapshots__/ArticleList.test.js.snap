// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView
    ListFooterComponent={[Function]}
    ListHeaderComponent={[Function]}
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    invertStickyHeaders={undefined}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={1}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onRefresh={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    refreshControl={
      <RefreshControlMock
        onRefresh={[Function]}
        progressViewOffset={undefined}
        refreshing={true}
      />
    }
    refreshing={true}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <RCTRefreshControl />
    <View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "#393e42",
                "borderBottomColor": "#000",
                "borderBottomWidth": 1,
                "borderTopColor": "#000",
                "borderTopWidth": 1,
              },
              Object {
                "backgroundColor": "#e1e8ee",
                "borderBottomColor": "#e1e1e1",
                "borderTopColor": "#e1e1e1",
              },
              undefined,
            ]
          }
        >
          <TextInput
            allowFontScaling={true}
            onChangeText={[Function]}
            onClearText={[Function]}
            placeholder="Suche nach einem Artikel..."
            placeholderTextColor="#86939e"
            style={
              Array [
                Object {
                  "backgroundColor": "#303337",
                  "borderRadius": 3,
                  "color": "#86939e",
                  "fontSize": 17.5,
                  "height": 30,
                  "margin": 8,
                  "overflow": "hidden",
                  "paddingLeft": 26,
                  "paddingRight": 19,
                },
                Object {
                  "backgroundColor": "#bdc6cf",
                },
                false,
                false,
                undefined,
                false,
                Object {
                  "paddingRight": 30,
                },
              ]
            }
            underlineColorAndroid="transparent"
            value=""
          />
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            style={
              Array [
                Object {
                  "color": "#86939e",
                  "fontSize": 16,
                },
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "position": "absolute",
                    "top": 15.5,
                  },
                  Object {
                    "left": 16,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
          <Text
            accessible={true}
            allowFontScaling={false}
            ellipsizeMode="tail"
            onPress={[Function]}
            style={
              Array [
                Object {
                  "color": "#86939e",
                  "fontSize": 16,
                },
                Array [
                  Object {
                    "backgroundColor": "transparent",
                    "position": "absolute",
                    "top": 15.5,
                  },
                  Object {
                    "right": 16,
                  },
                  undefined,
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
              ]
            }
          >
            
          </Text>
        </View>
      </View>
      <View
        onLayout={[Function]}
        style={null}
      >
        <View
          style={
            Object {
              "borderColor": "#CED0CE",
              "borderTopWidth": 1,
              "paddingVertical": 20,
            }
          }
        >
          <ActivityIndicator
            animating={true}
            color="#F44336"
            hidesWhenStopped={true}
            size="large"
          />
        </View>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
